proxy_server/proxy_server.go

```diff
@@ -0,0 +1,102 @@
+// this works
+package main
+
+import "fmt"
+import "net/http"
+import "log"
+import "io/ioutil"
+import "bytes"
+import "strconv"
+import "strings"
+import "time"
+import "crypto/hmac"
+import "crypto/sha256"
+import "encoding/base64"
+import "encoding/json"
```

Comment:
we can use the other style of import that looks like a function call

Reply:
done